non-critical-infra/hosts/hetzner-01/default.nix

```diff
@@ -0,0 +1,64 @@
+{ config, pkgs, lib, inputs, ... }:
+
+{
+  imports =
+    [
+      ./hardware.nix
+      inputs.srvos.nixosModules.server
+      inputs.srvos.nixosModules.hardware-hetzner-online-amd
+    ];
+
+
+  deployment = {
+    targetHost = "65.109.26.213";
+    targetUser = "root";
+  };
+
+  # Bootloader.
+  boot.loader.grub.enable = true;
```

Comment:
A bit of a shame to be using grub. Does the box not support UEFI boot (systemd-boot)? (Not that it's worth changing now, I suppose.)

Reply:
I think these boxes are using bios by default, and uefi could be activated if we ask.

Reply:
Newer machines now have uefi by default on hetzner.